quickwit/quickwit-proto/src/lib.rs

```diff
@@ -252,3 +252,19 @@ impl search::SortOrder {
 }
 
 impl quickwit_common::pubsub::Event for ReportSplitsRequest {}
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize)]
```

Comment:
Let's move that to ` quickwit_proto::indexing`.

Reply:
done